src/mgmt/srv_query.c

```diff
@@ -744,13 +697,11 @@ ds_mgmt_dev_replace(uuid_t old_dev_uuid, uuid_t new_dev_uuid,
 	}
 
 	/* BIO device state after reintegration should be NORMAL */
-	strncpy(resp->dev_state, smd_dev_stat2str(SMD_DEV_NORMAL),
-		buflen);
+	resp->dev_state = NVME_DEV_FL_INUSE | NVME_DEV_FL_PLUGGED;
```

Comment:
I wonder if it might be useful to have the combinations of flags defined in the C code. Then anyone who wants to set the dev_state to "normal" will always have the right combo, and they can be imported into the Go.

Reply:
I was thinking about this but decided to keep the number of C definitions to the minimum needed and derive anything that was needed in the go code (the combinations are only used once in C and I think it's pretty clear as is), Again if I have to re-push will reconsider.

Reply:
Okay, that works for me.